samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -71,12 +73,10 @@ public ExecutionPlan plan(OperatorSpecGraph specGraph) {
     JobGraph jobGraph = createJobGraph(specGraph);
 
     // fetch the external streams partition info
-    updateExistingPartitions(jobGraph, streamManager);
+    fetchInputAndOutputStreamPartitions(jobGraph, streamManager);
```

Comment:
thanks for the rename! i like the distinction between "fetching" vs "computing" partition-counts

Reply:
I'm glad you like it :) You're most welcome.